packages/app/src/cli/commands/app/generate/extension.ts

```diff
@@ -138,72 +123,57 @@ export default class AppScaffoldExtension extends Command {
       reset: flags.reset,
     })
 
-    const {extensionType, extensionFlavor} = promptAnswers
+    const {extensionType, extensionFlavor, name} = promptAnswers
```

Comment:
I think I've brought this up in another PR, but this command contains a log of logic so I recommend extracting it into a service. It's fine if we do it in a follow-up PR.